migrations/20201004075522-add-organization.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
let's name migrations that add tables with the format `create-[model]`
so, `create-organizations`